.github/workflows/scale-test-gce.yaml

```diff
@@ -0,0 +1,230 @@
+name: Scale Test GCE
+
+on:
+  schedule:
+    - cron: '39 0 * * 1,4'
+  push:
```

Comment:
I think this trigger needs to be removed before merging.

Reply:
For sure, I left that in for testing. I just removed it.